tests/Aspire.Hosting.RabbitMQ.Tests/RabbitMQFunctionalTests.cs

```diff
@@ -88,11 +90,13 @@ public async Task WithDataShouldPersistStateBetweenUsages(bool useVolume)
                 {
                     var hb = Host.CreateApplicationBuilder();
                     hb.Configuration[$"ConnectionStrings:{rabbitMQ1.Resource.Name}"] = await rabbitMQ1.Resource.ConnectionStringExpression.GetValueAsync(default);
+                    hb.Services.AddXunitLogging(testOutputHelper);
                     hb.AddRabbitMQClient(rabbitMQ1.Resource.Name);
 
                     using (var host = hb.Build())
                     {
                         await host.StartAsync();
+                        await app.WaitForTextAsync($"Time to start RabbitMQ:", resourceName: "rabbitMQ").WaitAsync(TimeSpan.FromMinutes(1));
```

Comment:
```suggestion
                        await app.WaitForTextAsync($"Time to start RabbitMQ:", resourceName: rabbitMQ1.Resource.Name).WaitAsync(TimeSpan.FromMinutes(1));
```

But fine in a separate PR if you don't want to go through another build (do you take notes? ;)).